pv_site_production/app.py

```diff
@@ -18,6 +18,10 @@
 from pv_site_production.utils.config import load_config
 from pv_site_production.utils.imports import import_from_module
 
+logging.basicConfig(
+    level=getattr(logging, os.getenv("LOGLEVEL", "DEBUG")),
```

Comment:
We could consider using a click argument for the log level, but this is fine too.

Reply:
yea, ill leave this for the moment